packages/lib/models/settings/builtInMetadata.ts

```diff
@@ -1816,6 +1819,17 @@ const builtInMetadata = (Setting: typeof SettingType) => {
 			},
 		},
 
+		'voiceTyping.prompt': {
+			value: '',
```

Comment:
```suggestion
		'voiceTyping.prompt': {
			advanced: true,
			value: '',
```

It may make sense to move this to the advanced settings section by default.